### PR TITLE
docs: Minor wording change

### PR DESCRIPTION
**Pull request by x11x**
Changed "one of" to "among" because two things are discussed. Also tried to reword it so that it is clearer

<small>By submitting this pull request, I confirm that my contribution is made under the terms of the [MIT license](https://github.com/dafny-lang/dafny/blob/master/LICENSE.txt).</small>

### CODE REVIEW

File: docs/OnlineTutorial/guide.md [hunk 1]
```diff
@@ -1823,10 +1823,10 @@ to their `reads` cousins,
 except they say what can be changed, rather than what the value of the function
 depends on. In combination with reads, modification
 restrictions allow Dafny to prove properties of code that would otherwise be
-very difficult or impossible. `reads` and `modifies` are one of the tools that
-allow Dafny to work on one method at a time, because they restrict what would
-otherwise be arbitrary modifications of memory to something that Dafny can
-reason about.
+very difficult or impossible. `reads` and `modifies` are among the tools that
+make it possible for Dafny to work on one method at a time, because they
```
Comment (fabiomadge) [inline docs/OnlineTutorial/guide.md:1827]: Not sure this is better.

**Comment by fabiomadge** [comment]
Thanks for your suggestions! Unless you can _allow_ is unclear, I'll only merge the first one.

**Comment by x11x** [comment]
Sorry for the delay. "Allow" is fine, not sure why I changed that.
Also, I realized the sense of the original sentence was that "reads and modifies are [together/combined] one of the tools that....", maybe you prefer something like that?
This is not a huge issue, it makes sense even as it was originally, it just hit my ear strange when I read it.

**Comment by fabiomadge** [comment]
No worries, I just wanted to avoid surprising you by merging something that was diverging from you original intention. How about now?

**Review by fabiomadge — APPROVED** [review]

**Comment by x11x** [comment]
Looks good

**Pull request merged**